code/modules/mob/living/carbon/human/life.dm

```diff
@@ -78,6 +78,8 @@
 		handle_stamina()
 
 		if (is_diona())
+			if(health < 60)
+				diona_split_into_nymphs()
```

Comment:
Not sure if I like this, as I said. It's relatively needless, since we already split properly at death. And will add complexity when it can be avoided.

Reply:
What would be a good way to do it then? The point of this is to make sure nymphs no matter what have at least 20% of their health, so 60 health remaining / the 6 nymphs is 10 health(20% of their health the pr)